ReadMe.md

```diff
@@ -0,0 +1,20 @@
+# React DOM Lite
+
+_"Perfection is achieved, not when there is nothing more to add, but when there is nothing left to take away.” - Antoine de Saint-Exupery._
+
+Compliance and amazing cross browser support led to a robust but heavy `react-dom`. React DOM Lite is an attempt to sculpt away some of the mass away and see if we can make something more low powered device friendly.
+
+By no means, this is a pursuit for the perfect library. It's about finding out what it is that we can live without in `react-dom`?
```

Comment:
i think i'd just remove this line, its a bit awkward and doesn't add much.

Reply:
Sure.